### ISSUE TITLE
Accessibility issue: Heading levels should only increase by one on /11-support-sessions

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#6]
## What
An accessibility scan flagged the element `<h3 class="fw-bold mt-0 mb-3 card-title text-start h4">
                      <a href="https://outlook.office.com/book/CampusWebServices11Support@emailarizona.onmicrosoft.com/s/D76aJbk6EEmuRnQOrvEzog2" class="text-reset">Portfolio 1: Joe Gallegos</a>
                  </h3>` because heading levels should only increase by one. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/heading-order?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/061f8e6e-c827-47dd-bbcb-0142ed5752a5.png)

To fix this, ensure the order of headings is semantically correct.

Specifically:

Fix any of the following:
-   Heading order invalid

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
